### ISSUE TITLE
opm-grid broken?

### ISSUE THREAD

**Issue by GitPaean** [OPM/opm-grid#224]
Anyone else also see this?

```
153 In file included from /home/kaib/OPM-devel/debug/opm-grid/dune/grid/cpgrid/PersistentContainer.hpp:10:0,
154                  from /home/kaib/OPM-devel/debug/opm-grid/dune/grid/CpGrid.hpp:1183,
155                  from /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.hpp:25,
156                  from /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:30:
157 /home/kaib/local/dune2.3.1/release_mpi/include/dune/grid/utility/persistentcontainervector.hh: In constructor ‘Dune::PersistentContainerVector<G, IndexSet, Vector>::PersistentContainerVector(const IndexSet&, int, const Value&, const Allocator&)’:
158 /home/kaib/local/dune2.3.1/release_mpi/include/dune/grid/utility/persistentcontainervector.hh:33:7: warning: declaration of ‘indexSet’ shadows a member of 'this' [-Wshadow]
159        : codim_( codim ),
160        ^
161 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp: In function ‘void Dune::cpgrid::getNullEdgeList(void*, int, int, int, ZOLTAN_ID_PTR, ZOLTAN_ID_PTR, int*, ZOLTAN_ID_PTR, int*, int, float*, int*)’:
162 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:161:6: warning: parameter ‘err’ set but not used [-Wunused-but-set-parameter]
163  void getNullEdgeList(void *cpGridPointer, int sizeGID, int sizeLID,
164       ^
165 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp: In constructor ‘Dune::cpgrid::CombinedGridWellGraph::CombinedGridWellGraph(const Dune::CpGrid&, Opm::EclipseStateConstPtr, const double*, bool)’:
166 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:348:17: error: ‘WellConstPtr’ is not a member of ‘Opm’
167      std::vector<Opm::WellConstPtr> wells  = eclipseState->getSchedule()->getWells();
168                  ^
169 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:348:17: error: ‘WellConstPtr’ is not a member of ‘Opm’
170 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:348:34: error: template argument 1 is invalid
171      std::vector<Opm::WellConstPtr> wells  = eclipseState->getSchedule()->getWells();
172                                   ^
173 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:348:34: error: template argument 2 is invalid
174 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:348:43: error: invalid type in declaration before ‘=’ token
175      std::vector<Opm::WellConstPtr> wells  = eclipseState->getSchedule()->getWells();
176                                            ^
177 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:348:83: error: cannot convert ‘std::vector<const Opm::Well*>’ to ‘int’ in initialization
178      std::vector<Opm::WellConstPtr> wells  = eclipseState->getSchedule()->getWells();
179                                                                                    ^
180 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:356:31: error: request for member ‘begin’ in ‘wells’, which is of non-class type ‘int’
181      for (auto wellIter= wells.begin(); wellIter != wells.end(); ++wellIter) {
182                                ^
183 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:356:58: error: request for member ‘end’ in ‘wells’, which is of non-class type ‘int’
184      for (auto wellIter= wells.begin(); wellIter != wells.end(); ++wellIter) {
185                                                           ^
186 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:357:9: error: ‘WellConstPtr’ is not a member of ‘Opm’
187          Opm::WellConstPtr well = (*wellIter);
188          ^
189 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:357:27: error: expected ‘;’ before ‘well’
190          Opm::WellConstPtr well = (*wellIter);
191                            ^
192 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:359:52: error: ‘well’ was not declared in this scope
193          Opm::CompletionSetConstPtr completionSet = well->getCompletions(last_time_step);
194                                                     ^
195 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp: In member function ‘void Dune::cpgrid::CombinedGridWellGraph::postProcessPartitioningForWells(std::vector<int>&)’:
196 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:385:17: error: ‘WellConstPtr’ is not a member of ‘Opm’
197      std::vector<Opm::WellConstPtr> wells  = eclipseState_->getSchedule()->getWells();
198                  ^
199 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:385:17: error: ‘WellConstPtr’ is not a member of ‘Opm’
200 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:385:34: error: template argument 1 is invalid
201      std::vector<Opm::WellConstPtr> wells  = eclipseState_->getSchedule()->getWells();
202                                   ^
203 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:385:34: error: template argument 2 is invalid
204 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:385:43: error: invalid type in declaration before ‘=’ token
205      std::vector<Opm::WellConstPtr> wells  = eclipseState_->getSchedule()->getWells();
206                                            ^
207 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:385:84: error: cannot convert ‘std::vector<const Opm::Well*>’ to ‘int’ in initialization
208      std::vector<Opm::WellConstPtr> wells  = eclipseState_->getSchedule()->getWells();
209                                                                                     ^
210 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:395:31: error: request for member ‘begin’ in ‘wells’, which is of non-class type ‘int’
211      for (auto wellIter= wells.begin(); wellIter != wells.end(); ++wellIter) {
212                                ^
213 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:395:58: error: request for member ‘end’ in ‘wells’, which is of non-class type ‘int’
214      for (auto wellIter= wells.begin(); wellIter != wells.end(); ++wellIter) {
215                                                           ^
216 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:396:9: error: ‘WellConstPtr’ is not a member of ‘Opm’
217          Opm::WellConstPtr well = (*wellIter);
218          ^
219 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:396:27: error: expected ‘;’ before ‘well’
220          Opm::WellConstPtr well = (*wellIter);
221                            ^
222 /home/kaib/OPM-devel/debug/opm-grid/dune/grid/common/ZoltanGraphFunctions.cpp:398:52: error: ‘well’ was not declared in this scope
```

**Comment by bska** [comment]
I don't `HAVE_ZOLTAN` so I can't reproduce this symptom.

**Comment by GitPaean** [comment]
Looks like related to the depreciated `WellConstPtr`.

**Comment by GitPaean** [comment]
fixed #225, closing now.
